wordpress-mobile/WordPress-Android
# Fix setting post ID for media uploads

**Not ready to merge!**

~~https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/426 should be merged first~~, and ~~3c2fb12 changed to the latest `develop` hash~~ done in 71cccab.

Sets the parent post ID of media uploads (when uploaded to a post that already exists remotely), and fixes a case where we were incorrectly setting the _local_ post ID as the media's parent post ID.

To test:
* Use the editor to upload media to posts (WP.com and self-hosted, local drafts and remote posts)
* After upload, open the media in the WP Media Library on the browser and confirm that:
  1. Media that were added to existing remote posts have those posts set as their parent (`Uploaded To` field)
  2. Media that were added to local draft posts have no parent
